content/en/events/guides/migrating_to_new_events_features.md

```diff
@@ -6,69 +6,86 @@ aliases:
 ---
 
 <div class="alert alert-warning">
-  Use this guide to migrate from the legacy Events Stream to the Events Explorer. The Events Explorer is being automatically rolled out to all customers in Q1 and Q2 2022. If you see the <strong>Explorer</strong> option under <strong>Events</strong> in the left navigation bar, you have the new features. For legacy events information, see the <a href="/events/stream/">Event Stream</a>.</div>
+  On <strong>June 30, 2022</strong>, we will retire Datadog's existing event stream and event monitors. We are migrating all customers to a new and improved events experience. This page contains important information about this migration and steps to take before the retirement date to ensure that your existing event visualizations and monitors continue to work properly.</div>
 
-The Events Explorer replaces the Event Stream, one of Datadog's oldest features. The new Event query syntax requires migration of Event monitors, dashboard widgets, and any other features that query Events. 
+## Table of Contents
+- [Why the migration?](#why-the-migration)
+- [What is the migration timeline?](#what-is-the-migration-timeline)
+- [What action do I need to take?](#what-action-do-i-need-to-take)
+- [What's new?](#whats-new)
+- [What changed?](#what-changed)
+- [Examples](#examples)
 
-The introduction of the Events Explorer and its associated changes means that Event features work similarly to other Datadog products. Event query syntax, explorer and analytics views, processing pipelines and other features look like their counterparts in Logs or APM Traces. See [Events Explorer][1] for more details.
+## Why the migration?
 
-## What this means for your organization
+Launched more than 10 years ago, Datadog's event stream is one of our earliest features. Our new events experience includes many new features that allow you to get even more value from your events. These include event analytics, the ability to generate metrics from your events, and a much friendlier and more intuitive query syntax that is better aligned with other Datadog products, such as Log Management and APM.
```

Comment:
It also includes post processing through pipelines

Reply:
Added and added a section in the new changes